libretroshare/src/util/rsthreads.cc

```diff
@@ -103,6 +104,15 @@ RsThread::RsThread()
     mShouldStopSemaphore.set(0) ;
 }
 
+RsThread::~RsThread()
+{
+	if(isRunning())
+    {
+		RsErr() << "Deleting a thread that is still running! Something is very wrong here and Retroshare is likely to crash because of this." << std::endl;
```

Comment:
please don't forget to put __PRETTY_FUNCTION__ at beginning of the message, it will be easier to understand from where the error come from, in 6 months from now ;)

Reply:
Well add PRETTY_FUNCTION inside the macro, so as to avoid adding this everywhere.